docs/ref/forms/formsets.txt

```diff
@@ -1,18 +1,78 @@
-=================
-Formset Functions
-=================
+=======
+Formset
+=======
 
 Formset API reference. For introductory material about formsets, see the
 :doc:`/topics/forms/formsets` topic guide.
 
 .. module:: django.forms.formsets
    :synopsis: Django's functions for building formsets.
 
+``Making a formset``
+====================
+
+.. note::
+    From 3.0 and onwards, a formset can be made using two ways either
+    by using the `formset_factory` or by using the `FormSet` class both are
+    mentioned below.
+
+
+``FormSet class``
+-----------------
+
+.. class:: FormSet.objects
+
+    The :class:`~django.forms.formsets.FormSet` class is used to create the
+    ``Formset`` from the given ``form`` using a declarative syntax.
+    See the following example which shows how to create AuthorFormSet from
+    AuthorForm::
+
+        from django.forms import formsets
+
+        class AuthorFormSet(formsets.FormSet):
+            # adding a form field is must
+            form = AuthorForm
+
 ``formset_factory``
-===================
+-------------------
 
 .. function:: formset_factory(form, formset=BaseFormSet, extra=1, can_order=False, can_delete=False, max_num=None, validate_max=False, min_num=None, validate_min=False)
 
     Returns a ``FormSet`` class for the given ``form`` class.
 
     See :doc:`formsets </topics/forms/formsets>` for example usage.
+
+``Overriding methods in fromset``
```

Comment:
`FormSet`

Reply:
So what you are saying is 

Move the explanation stuff, like two methods to create a Formset in topics and keeping just the definitions and short explanations in ref?

Correct me if I'm wrong, becuase I'm working in that direction